### PR TITLE
Refactor to use Apache on Centos 7

### PR DESCRIPTION
**Pull request by gchaix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +27,7 @@ For full details, visit https://www.dokuwiki.org/dokuwiki
 
 ## Customizations
 
-The base container starts with the Alpine distribution, adding lighttpd and
+The base container starts with the Centos 7 distribution, adding Apahe and
```
Comment (jeffsheltren) [inline README.md]: Typo, should be: `Apache`

File: README.md [hunk 2]
```diff
@@ -40,14 +41,14 @@ You can comment out the line in Dockerfile that installs `php7-curl` and
...
-To change which port lighttpd listens on, edit `lighttpd/lighttpd.conf` changing:
+To change which port Apache listens on, edit `app/apache.conf` changing:
 ```
-	server.port          = 8080
+	<VirtualHost *:80>
 ```
-from `8080` to your desired port. Then, also edit the `EXPOSE` toward the end of
-Dockerfile, again changing `8080` to your new port.
+from `80` to your desired port. Then, also edit the `EXPOSE` toward the end of
+Dockerfile, again changing `808 to your new port.
```
Comment (jeffsheltren) [inline README.md]: `80` ?

File: app/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,52 @@
+FROM centos:7
+
+ENV PATH=$PATH:/opt/rh/php71u/root/usr/bin
```
Comment (jeffsheltren) [inline app/Dockerfile]: That /opt path shouldn't be needed since that's from SCL. Can we do PHP 7.2 or 7.3 instead?

**Comment by gchaix** [comment]
Ignore the `./helm/*` for now.  It was being overly-complicated.  The correct yaml to use is in `./yaml`.

File: Dockerfile [hunk 4]
```diff
@@ -1,30 +1,53 @@
-FROM	alpine:3.9
-
+FROM centos:7
+
+ENV PATH=$PATH:/opt/rh/php72u/root/usr/bin
```
Comment (jeffsheltren) [inline Dockerfile]: `/opt/rh/php72u/root/usr/bin` shouldn't be needed? IUS packages don't install into `/opt` AFAIK

File: Dockerfile [hunk 5]
```diff
@@ -1,30 +1,53 @@
...
+
+RUN yum install -y epel-release && yum install -y https://$(rpm -E '%{?centos:centos}%{!?centos:rhel}%{rhel}').iuscommunity.org/ius-release.rpm && \
+    INSTALL_PKGS="php72u php72u-cli php72u-mysqlnd php72u-pgsql php72u-bcmath \
+                  php72u-gd php72u-intl php72u-ldap php72u-mbstring php72u-pdo \
+                  php72u-process php72u-soap php72u-opcache php72u-xml \
+                  php72u-gmp php72u-pecl-apcu php72u-pecl-memcached \
+                  php72u-xml php72u-mbstring \
+                  php72u-gd php72u-zip php72u-mcrypt \
+                  wget curl git unzip yum-plugin-security" && \
```
Comment (jeffsheltren) [inline Dockerfile]: Why `yum-plugin-security`? I don't that gets us much in a container.

Comment (jeffsheltren) [reply Dockerfile]: Oh, I see now the yum call below.. not sure how I feel about it, but not a blocker. Let's leave it while I ponder life, the universe, and gods of thunder.

**Review by jeffsheltren — APPROVED** [review]

**Pull request merged**